frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance_view.dart

```diff
@@ -394,17 +411,58 @@ class _ThemeFontFamilySettingState extends State<ThemeFontFamilySetting> {
   }
 }
 
+class ThemeSettingEntry extends StatelessWidget {
+  const ThemeSettingEntry({
+    super.key,
+    this.resetButtonKey,
+    required this.label,
+    this.trailing,
+    this.onResetRequested,
+  });
+
+  final String label;
+  final Key? resetButtonKey;
+  final List<Widget>? trailing;
+  final void Function()? onResetRequested;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Expanded(
+          child: FlowyText.medium(
+            label,
+            overflow: TextOverflow.ellipsis,
+          ),
+        ),
+        if (trailing != null) ...trailing!,
+        if (onResetRequested != null)
+          FlowyIconButton(
+            key: resetButtonKey,
+            width: 24,
+            icon: FlowySvg(
```

Comment:
`icon: const FlowySvg(
              name: 'common/recover',
            ),`
            
I think by default the onBackground color is applied, but please confirm this on your system.

Reply:
Looks like `svgWidget` uses `Theme.of(context).iconTheme.color`. I'm not sure if it's the same, but I'm going to leave it as is.